build.go

```diff
@@ -0,0 +1,45 @@
+package static
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/paketo-buildpacks/nginx"
+	"github.com/paketo-buildpacks/packit/v2"
+	"github.com/paketo-buildpacks/packit/v2/scribe"
+)
+
+func Build(logger scribe.Emitter) packit.BuildFunc {
+	return func(context packit.BuildContext) (packit.BuildResult, error) {
+		var webRoot string
+		for _, entry := range context.Plan.Entries {
+			if v, ok := entry.Metadata[webRootKey]; ok {
+				webRoot, ok = v.(string)
+				if !ok {
+					return packit.BuildResult{}, fmt.Errorf("%s in metadata is not a string", webRootKey)
+				}
```

Comment:
I think once we found the `webRoot` we can exit the for loop?

Reply:
done